plugins/module_utils/models.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
this can be removed as we are going to support python3+ only for this collection